cli/shared/src/main/scala/com/lightbend/rp/reactivecli/http/Http.scala

```diff
@@ -44,12 +46,22 @@ object Http {
 
           val location = response.headers("Location")
 
-          if (visitedUrls.contains(location) || visitedUrls.length >= settings.maxRedirects)
+          if (visitedUrls.contains(location) || visitedUrls.length >= settings.maxRedirects) {
+            logger.debug("No more redirects allowed")
             Future.failed(InfiniteRedirect(visitedUrls))
-          else
-            doRequest(
-              HttpRequest(location).copy(requestFollowRedirects = request.requestFollowRedirects),
-              location :: visitedUrls)
+          } else {
+            println(location)
+            if (location.indexOf("X-Amz-Credential") > -1) {
```

Comment:
In general, could you split this change to another commit with a message describing what it does please?
What you wrote in the PR description is fine.